addons/sale_service/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
# -*- coding: utf-8 -*-
# Part of Odoo. See LICENSE file for full copyright and licensing details.
```
😇 
In general, I try to put it in every python files. 🙂

Reply:
Sorry but it is not done. 😬

Reply:
Ooops.. It is done now :+1: